modules/aws/eks/eks.tf

```diff
@@ -33,10 +34,12 @@ data "aws_subnet" "cluster" {
 resource "aws_eks_cluster" "this" {
   provider = aws.eks_admin
 
-  name     = "${var.environment}-${var.name}${var.eks_name_suffix}"
+  name     = local.eks_cluster_name
   role_arn = var.cluster_role_arn
   version  = var.eks_config.kubernetes_version
 
+  enabled_cluster_log_types = ["api", "audit"]
```

Comment:
Do we want to make this into to a variable to make it easy to remove/add specific logs that you are interested in?

Reply:
What else is available?

Reply:
https://docs.aws.amazon.com/eks/latest/userguide/control-plane-logs.html

- Kubernetes API server component logs (api) 
- Audit
- authenticator
- Controller manager (controllerManager)
- Scheduler (scheduler)